.github/workflows/node.js.yml

```diff
@@ -55,6 +55,26 @@ jobs:
             - run: npm run testCompile
             - run: npm run lint
 
+    jscpd:
+        # needs: lint-commits
```

Comment:
looks like we just need to remove this an its probably good to merge?

Reply:
Yeah, still exploring adding some more functionality, but that can be follow up. Seems pretty low-risk to test this simple version in the meantime.